lib/humanize/core_ext.rb

```diff
@@ -0,0 +1,20 @@
+module Humanize
+  def humanize(locale: Humanize.config.default_locale,
+               decimals_as: Humanize.config.decimals_as)
```

Comment:
This is the glue code that converts from implicit `self` arg to explicit `self` passed as the `number` arg.